spring-batch-docs/asciidoc/scalability.adoc

```diff
@@ -260,30 +263,30 @@ Spring Batch creates step executions for the partitions called
 The `PartitionHandler` is the component that
       knows about the fabric of the remoting or grid environment. It is able
       to send `StepExecution` requests to the remote
-      `Steps`, wrapped in some fabric-specific format, like a DTO. It does not
-      have to know how to split up the input data, or how to aggregate the
-      result of multiple `Step` executions. Generally speaking it probably also
-      doesn't need to know about resilience or failover, since those are
-      features of the fabric in many cases, and anyway Spring Batch always
-      provides restartability independent of the fabric: a failed `Job` can
-      always be restarted and only the failed `Steps` will be
+      `Step` implementations, wrapped in some fabric-specific format, like a DTO. It does not
```

Comment:
`instances` instead of `implementations`

Reply:
Yes, please.